src/VolleyManagement.Backend/VolleyManagement.Data.MsSql/Repositories/TournamentRepository.cs

```diff
@@ -11,15 +11,13 @@
     using Mappers;
     using Specifications;
 
+    using static VolleyManagement.Data.MsSql.Mappers.DomainToDal;
```

Comment:
We don't need it here. We need to be explicit

Reply:
ok